src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img src="./images/logo.png"
+             alt="logo" class="logo">
```

Comment:
please fix the code styling according to the checklist
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/cd66e129-eeae-43a2-aeed-8eb9adce8365)